src/communication/azext_communication/manual/_params.py

```diff
@@ -8,17 +8,147 @@ def load_arguments(self, _):
     with self.argument_context('communication update') as c:
         c.argument('location', validator=None)
 
+    _load_identity_arguments(self)
+    _load_sms_arguments(self)
+    _load_phonenumber_arguments(self)
+    _load_chat_arguments(self)
+
+
+def _load_identity_arguments(self):
+    self.argument_context('communication identity create-user')
+
+    with self.argument_context('communication identity delete-user') as c:
+        c.argument('user_id', options_list=['--user-id', '-u'], type=str, help='ACS identifier')
```

Comment:
Please remove `-id` suffix in argument option name. It's not recommended as well. Unless they are already released.